### PR TITLE
(feat) [SD-3626] Modified the related item widget based on the pharse prefix query.

### PR DESCRIPTION
**Pull request by mdhaman**
This PR is dependent on [CORE-PR](https://github.com/superdesk/superdesk-core/pull/80)

This PR requires following command:
`python manage.py app:rebuild_elastic_index`

Or

Drop the Superdesk Index and Stop-Start the backend and it should create new empty Superdesk index.
Then run the following commands.
`python manage.py app:index_from_mongo --from ingest`
`python manage.py app:index_from_mongo --from archive`
`python manage.py app:index_from_mongo --from published`

### CODE REVIEW

**Comment by sivakuna-aap** [comment]
:+1:

**Comment by mugurrus** [comment]
:+1:

File: client/app/scripts/superdesk/itemList/itemList.js [hunk 1]
```diff
@@ -113,24 +113,10 @@ angular.module('superdesk.itemList', ['superdesk.search'])
...
-            queryRelatedItem.push('slugline' + ':("' + options.keyword + '")');
-
-            for (var i = 0; i < length; i++) {
-                if (options.keyword) {
-                    queryRelatedItem.push('slugline' + ':(' + queryWords[i] + ')');
-                }
-            }
-
-            if (queryRelatedItem.length) {
+            if (options.keyword) {
```
Comment (plamut) [inline client/app/scripts/superdesk/itemList/itemList.js]: We can combine this into a single `if` statement, saving a level of (unnecessary) indentation.

**Comment by plamut** [comment]
:+1:  (with just a minor non-significant remark)

**Comment by mdhaman** [comment]
Dependent on the https://github.com/superdesk/superdesk/pull/1376 to be merged.

**Pull request merged**